### ISSUE TITLE
Unicode characters in category names

### ISSUE THREAD

**Issue by rob-carruthers** [aldanor/pynab#1]
£ sign in category names is causing an error in category handling.
Removing the £ sign from category names is a workaround.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/ynab/models.py", line 326, in __str__
    return str(self._elements)
  File "/usr/lib64/python2.7/site-packages/ynab/models.py", line 248, in __repr__
    info += ' ({})'.format(self.category.name)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa3' in position 16: ordinal not in range(128)
```

**Comment by aldanor** [comment]
@grooveharder Thanks for reporting... yea, this is a known py2/py3 problem – Python 2 expects `__repr__` to return a byte string, whereas Python 3 expects a unicode object, hence all the problems.

Anyways, should be fixed now in a2aa455, let me know if it doesn't work :)

**Comment by rob-carruthers** [comment]
Ah that's why, on py2 over here. Fixed in latest pull, thanks.
